tests/Oryx.Tests.Common/ImageTestHelper.cs

```diff
@@ -50,16 +50,98 @@ public ImageTestHelper(ITestOutputHelper output)
 
         /// <summary>
         /// Constructs a runtime image from the given parameters that follows the format
-        /// {image}/{platform}:{platformVersion}. Since no image is provided, the default image 'oryxdevmcr.azurecr.io',
-        /// or any image set by environment variable will be used. If any tag suffix was set as an environment variable,
-        /// it will be appended to the end of the tag.
+        /// '{image}/{platformName}:{platformVersion}{tagSuffix}'. The base image can be set with the environment
+        /// variable ORYX_TEST_IMAGE_BASE, otherwise the default base 'oryxdevmcr.azurecr.io/public/oryx' will be used.
+        /// If a tag suffix was set with the environment variable ORYX_TEST_TAG_SUFFIX, it will be appended to the tag.
         /// </summary>
-        /// <param name="platform">The platform to pull the runtime image from.</param>
+        /// <param name="platformName">The platform to pull the runtime image from.</param>
         /// <param name="platformVersion">The version of the platform to pull the runtime image from.</param>
         /// <returns>A runtime image that can be pulled for testing.</returns>
-        public string GetRuntimeImage(string platform, string platformVersion)
+        public string GetTestRuntimeImage(string platformName, string platformVersion)
         {
-            return $"{_image}/{platform}:{platformVersion}{_tagSuffix}";
+            return $"{_image}/{platformName}:{platformVersion}{_tagSuffix}";
+        }
+
+        /// <summary>
+        /// Constructs a 'build' image using either the default image base (oryxdevmcr.azurecr.io/public/oryx), or the
+        /// base set by the ORYX_TEST_IMAGE_BASE environment variable. If a tag suffix was set with the environment
+        /// variable ORYX_TEST_TAG_SUFFIX, it will be used as the tag, otherwise, the 'latest' tag will be used.
+        /// </summary>
+        /// <returns>A 'build' image that can be pulled for testing.</returns>
+        public string GetTestBuildImage()
```

Comment:
I feel this class now has enough logic which warrants a unit test?

Reply:
Add a new suite of tests in `ImageTestHelperTest`.